tests/containers/bci_version_check.pm

```diff
@@ -42,6 +42,10 @@ sub run {
         die('No valid container engines defined in CONTAINER_RUNTIMES variable!');
     }
 
+    # Avoid unnecessary load on IBS by holding back all test runs except 15-SP7, so that registry.suse.de can load the images into the cache
+    # This is a temporary workaround until https://progress.opensuse.org/issues/189813 is done.
+    sleep(300) unless check_var('HOST_VERSION', '15-SP7');
```

Comment:
Adding a variable to skip this `sleep` when doing VR's would be nice.

Reply:
Sorry, I missed this comment but I see this was resolved in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/23487.